e2e-setup/mocks/git.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# hack to mock git commands as part of action.yaml so that we could simulate dags only deploy scenario without making any additional commits
```

Comment:
Should we first look at the args to see which git command was called? That would make it clearer how to mock more git commands in the future.